tutorials/interoperability/using-axelar-sdk.md

```diff
@@ -0,0 +1,226 @@
+
+The estimateGasFee function provided by the Axelar SDK will find the conversion between the origin chain’s native currency and the destination chain’s native currency to find the right amount to send to the destination chain.
+
+You, the astute reader, might wonder why we’re using `GLMR` instead of `DEV`. Similar to how Axelar uses the MainNet chain names instead of using the TestNet names, Axelar will interpret `GLMR` as `DEV` since we’re using the TestNet environment.
+
+```js
+--8<-- 'code/tutorials/interoperability/axelar-sdk/mint-4.js'
+```
+
+Calling this function from the SDK will return a string representing the amount of DEV WEI to pay, like `241760932800000`. That’s hard for us simple humans to understand, so we use Ethers.js to convert it into a more human-readable version to print to the console later.
+
+```js
+--8<-- 'code/tutorials/interoperability/axelar-sdk/mint-5.js'
+```
+
+Finally, we call the `mintXCNFT` contract function. The important takeaway here is that we’re sending the gas fee not as a gas limit but as value. Ethers.js can calculate how much gas to send on the origin chain. However, to pay for the destination chain, we have to calculate with the Axelar SDK and send it as value to the `IAxelarGasReceiver` contract.
+
+```js
+--8<-- 'code/tutorials/interoperability/axelar-sdk/mint-6.js'
+```
+
+That’s the entire script! Before we run the script, check again to make sure that the four constants (`ORIGIN_CHAIN`, `DESTINATION_CHAIN`, `ORIGIN_CHAIN_ADDRESS`, `DESTINATION_CHAIN_ADDRESS`) at the top of the script are set correctly.
+
+Here’s the command to mint your NFT!
+
+```bash
+npx hardhat run scripts/mint.js --network moonbase
+```
+
+The console should output something similar to this:
+
+--8<-- 'code/tutorials/interoperability/axelar-sdk/terminal/mint.md'
+
+The most important data here is the minting transaction because that’s how you track your transaction's status. So don’t lose it! But if you do, you can look at all of the recent transactions on [Axelar’s TestNet scanner](https://testnet.axelarscan.io/){target=\_blank}.
+
+## Viewing Axelar Transaction Status {: #viewing-axelar-transaction-status}
+
+Axelar has a [TestNet explorer](https://testnet.axelarscan.io/gmp/search){target=\_blank}, and a successful transaction for the interaction you just completed would look something like this:
+
+![Viewing Transaction status on AxelarScan](/images/tutorials/interoperability/axelar-sdk/axelar-2.webp)
+
+It's a good idea to try out the SDK to view the status of your transactions because it gives more information about your transaction and any possible errors. To do this, I wrote a Hardhat task for us to use. You can view the code in axelarStatus.js, but we’ll take a dive here too.
```

Comment:
```suggestion
It's a good idea to try out the SDK to view the status of your transactions because it gives more information about your transaction and any possible errors. To do this, I wrote a Hardhat task for us to use. You can view the code in `axelarStatus.js`, but we’ll take a dive here too.
```